openstack/compute/v2/servers/results.go

```diff
@@ -217,6 +217,10 @@ type Server struct {
 
 	// Fault contains failure information about a server.
 	Fault Fault `json:"fault"`
+
+	// Tags is a list of server tags. Tags are arbitrarily defined strings
+	// attached to a server.
+	Tags []string `json:"tags"`
```

Comment:
Per the other comments/discussion, please revert this as there's already a way to achieve this. Let me know if you have any questions with the current implementation, though.